### PR TITLE
Adds the updated managed_notebooks/predictive_maintenance notebook to the official folder 

### PR DESCRIPTION
**Pull request by krishr2d2**
Opening a PR for `Official Notebooks` under the [notebooks/official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/official) folder:
- [x] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [x] Follow the style and grammar rules outlined in the above notebook template.
- [x] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [x] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/docs/contributing.md#code-quality-checks).
- [ ] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [x] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/docs/CODEOWNERS) file under `# Official Notebooks` section, pointing to the author or the author's team.
- [x] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

### CODE REVIEW

**Comment by andrewferlitsch** [comment]
Failed due to:
Change filter did not match

**Comment by kweinmeister** [comment]
FYI @ivanmkc @andrewferlitsch `ResourceExhausted: 429 The following quotas are exceeded: DeployedCustomModelsPerProjectPerRegion`

File: notebooks/official/workbench/predictive_maintainance/predictive_maintenance_usecase.ipynb [hunk 1]
```diff
@@ -1,12 +1,57 @@
 {
```
Comment (andrewferlitsch) [inline notebooks/official/workbench/predictive_maintainance/predictive_maintenance_usecase.ipynb:1]: <p>Separate into 4 cells</p><p>Move Dataset after Objective</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/521/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (andrewferlitsch) [inline notebooks/official/workbench/predictive_maintainance/predictive_maintenance_usecase.ipynb:1]: <p>Google Cloud Notebooks -&gt; Vertex AI Workbench Notebooks</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/521/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Comment (andrewferlitsch) [inline notebooks/official/workbench/predictive_maintainance/predictive_maintenance_usecase.ipynb:1]: <p>set delete_bucket to False</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/521/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='76'/>

Comment (krishr2d2) [reply notebooks/official/workbench/predictive_maintainance/predictive_maintenance_usecase.ipynb:1]: done..!!

Comment (krishr2d2) [reply notebooks/official/workbench/predictive_maintainance/predictive_maintenance_usecase.ipynb:1]: Updated !!

Comment (krishr2d2) [reply notebooks/official/workbench/predictive_maintainance/predictive_maintenance_usecase.ipynb:1]: Done !!

**Review by andrewferlitsch — APPROVED** [review]

**Pull request merged**